### ISSUE TITLE
[bug] timeout 

### ISSUE THREAD

**Issue by florianlenz** [gnosis/MultiSigWallet#95]
Hi, 

we are playing around with the gnosis multi sig wallet. When we try to confirm a transaction with ledger we almost have no time to read the address, eth and gas feed since we are getting a timeout from the gnosis wallet. Could you please increase the timeout? 

Thank you.

**Comment by denisgranha** [comment]
Yeah would be nice, I remember it was harcoded in one of the dependent libraries, will try to include this in the next release

**Comment by florianlenz** [comment]
Thank you. Tell me if I can support you.

**Comment by jeremyschlatter** [comment]
It looks like this might be fixed upstream, as of https://github.com/LedgerHQ/ledgerjs/pull/67

Unfortunately it also looks to me like you can't simply bump the version number, because the [relevant package](https://www.npmjs.com/package/ledgerco) hasn't been updated in ten months. Ledger has a bunch of other npm packages that are up-to-date, but switching to them might require a non-trivial code rewrite.

I encountered this issue today and it was pretty frustrating.

**Comment by denisgranha** [comment]
one alternative is use the electron version (desktop build) it uses another ledger library that doesn’t have tiemout
